e2e/e2e.go

```diff
@@ -192,6 +197,70 @@ func main() {
 
 	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
 
+	// runEndToEndTests is a flag that denotes whether or not this needs to actually
+	// run or not based off of the filepath.Walk function immediately proceeding.
+	var runEndToEndTests bool
+
+	err := filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
+		if runEndToEndTests {
+			// No need to keep walking through files if we've already found one file
+			// that requries e2e tests.
+			return nil
+		}
+
+		if err != nil {
+			return err
+		}
+
+		if info.Mode()&os.ModeSymlink == os.ModeSymlink {
```

Comment:
Is this because we are still on go 1.15 and WalkDir was introduced in 1.16?

Reply:
yeah we're not completely switched over to 1.16 as a whole so i think it's safe to continue to use filepath.Walk